### PR TITLE
doc: enhance a string of problems to improve UE

### PR DESCRIPTION
**Pull request by aroundabout**
## Main change

- in hubble page(cn), title level of the `#### 2 部署` is wrong

### CODE REVIEW

File: content/cn/docs/quickstart/hugegraph-hubble.md
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-hubble.md:1]: need sync the EN version?

**Review by imbajin — COMMENTED** [review]
And also enhance the docker usage doc:
1. image
2. use format plugin to enhance the style automatically

File: content/cn/docs/SUMMARY.md [hunk 1]
```diff
@@ -5,11 +5,12 @@
...
 * [Install HugeGraph-Server](quickstart/hugegraph-server.md)
 * [Load data with HugeGraph-Loader](quickstart/hugegraph-loader.md)
-* [Manage with HugeGraph-Tools](quickstart/hugegraph-tools.md)
 * [Visual with HugeGraph-Hubble](quickstart/hugegraph-hubble.md)
-* [Display with HugeGraph-Studio](quickstart/hugegraph-studio.md)
 * [Develop with HugeGraph-Client](quickstart/hugegraph-client.md)
+* [Manage with HugeGraph-Tools](quickstart/hugegraph-tools.md)
 * [Analysis with HugeGraph-Computer](quickstart/hugegraph-computer.md)
+<!-- * [Display with HugeGraph-Studio](quickstart/hugegraph-studio.md) -->
+<!-- studio has been outdated -->
```
Comment (imbajin) [inline content/cn/docs/SUMMARY.md:13]: 暂时不能直接移除 studio, 让它处于一个最后 / outdated 的位置, 等下一个大版本 2.0 再考虑完全 hide

File: content/cn/docs/quickstart/hugegraph-server.md [hunk 2]
```diff
@@ -26,35 +26,37 @@ java -version
...
 
-有三种方式可以部署 HugeGraph-Server 组件：
+有四种方式可以部署 HugeGraph-Server 组件：
 
-- 方式 1：一键部署
+- 方式 1：使用 Docker 容器 (推荐)
 - 方式 2：下载 tar 包
 - 方式 3：源码编译
-- 方式 4：使用 Docker 容器
+- 方式 4：一键部署
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-server.md:34]:
```suggestion
- 方式 4：使用 tools 工具部署 (Outdated)
```

File: content/cn/docs/quickstart/hugegraph-server.md [hunk 3]
```diff
@@ -105,9 +107,28 @@ mvn package -DskipTests
 
 执行成功后，在 hugegraph 目录下生成 hugegraph-*.tar.gz 文件，就是编译生成的 tar 包。
 
-#### 3.4 使用 Docker 容器
 
-可参考 [Docker 部署方式](https://hub.docker.com/r/hugegraph/hugegraph)。
+#### 3.4 一键部署
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-server.md:111]: same

File: content/en/docs/SUMMARY.md
Comment (imbajin) [inline content/en/docs/SUMMARY.md:1]: 英文文档最好等 CN 固化了再更新, 免得反复调整

File: content/cn/docs/quickstart/hugegraph-server.md [hunk 4]
```diff
@@ -148,7 +215,13 @@ Connecting to HugeGraphServer (http://127.0.0.1:8080/graphs)....OK
 
 提示的 url 与 rest-server.properties 中配置的 restserver.url 一致
 
-#### 5.2 RocksDB
+</details>
+
+##### 5.2.2 RocksDB
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-server.md:220]: 改变一下后端排序: (按频率)
rocksdb - hbase - mysql - Cassandra - memory - ScyllaDB(或其他)

**Comment by imbajin** [comment]
@VGalaxies could also check the PR~

File: content/cn/docs/quickstart/hugegraph-hubble.md [hunk 5]
```diff
@@ -32,14 +32,40 @@ HugeGraph是一款面向分析型，支持批量操作的图数据库系统，
...
+我们可以使用 `docker run -itd --name=hubble -p 8088:8088 hugegraph/hubble` 快速启动 [hubble](https://hub.docker.com/r/hugegraph/hubble).
+
+或者使用docker-compose启动hubble，另外如果hubble和graph在同一个docker网络下，可以使用graph的contain_name进行访问，而不需要宿主机的ip
+
+使用`docker-compose up -d`，`docker-compose.yml`如下：
+
+```yaml
+version: '3'
+services:
+  graph_hubble:
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-hubble.md:55]:
```suggestion
  server:
```

so as EN version

File: content/en/docs/quickstart/hugegraph-hubble.md [hunk 6]
```diff
@@ -35,11 +35,38 @@ For Gremlin tasks that need to traverse the whole graph, index creation and reco
...
+We can use `docker run -itd --name=hubble -p 8088:8088 hugegraph/hubble` to quick start [hubble](https://hub.docker.com/r/hugegraph/hubble).
+
+Alternatively, you can use Docker Compose to start `hubble`. Additionally, if `hubble` and the graph are in the same Docker network, you can access the graph using the container name of the graph, eliminating the need for the host machine's IP address.
+
+Use `docker-compose up -d`，`docker-compose.yml` is following:
+
+```yaml
+version: '3'
+services:
+  graph_hubble:
```
Comment (aroundabout) [inline content/en/docs/quickstart/hugegraph-hubble.md:55]:
```suggestion
  graph_hubble:
  server:
```

File: content/cn/docs/quickstart/hugegraph-server.md [hunk 7]
```diff
@@ -117,6 +138,49 @@ mvn package -DskipTests
 
 ### 5 启动
 
+#### 5.1 使用docker
+
+在 [3.1 使用 Docker 容器](#31-使用-docker-容器-推荐) 中, 我们已经介绍了 如何使用 `docker` 部署 `hugegraph-server`, 我们还可以设置参数在sever启动的时候加载样例图
```
Comment (VGalaxies) [inline content/cn/docs/quickstart/hugegraph-server.md:143]: remove extra space between CJK words?

**Comment by VGalaxies** [comment]
Good job! Consider using [autocorrect](https://plugins.jetbrains.com/plugin/20244-autocorrect) plugin to improve the formatting. 🚀

File: content/cn/docs/quickstart/hugegraph-server.md [hunk 8]
```diff
@@ -462,125 +582,5 @@ $bin/stop-hugegraph.sh
...
-
-```javascript
-> curl "http://localhost:18080/graphs/hugegraph/graph/vertices" | gunzip
-
-{"vertices":[{"id":"2:lop","label":"software","type":"vertex","properties":{"name":"lop","lang":"java","price":328}},{"id":"1:josh","label":"person","type":"vertex","properties":{"name":"josh","age":32,"city":"Beijing"}},{"id":"1:marko","label":"person","type":"vertex","properties":{"name":"marko","age":29,"city":"Beijing"}},{"id":"1:peter","label":"person","type":"vertex","properties":{"name":"peter","age":35,"city":"Shanghai"}},{"id":"1:vadas","label":"person","type":"vertex","properties":{"name":"vadas","age":27,"city":"Hongkong"}},{"id":"2:ripple","label":"software","type":"vertex","properties":{"name":"ripple","lang":"java","price":199}}]}
-```
-
-代表创建示例图成功。
+  
+\
```
Comment (aroundabout) [inline content/cn/docs/quickstart/hugegraph-server.md:586]:
```suggestion
```

**Review by imbajin — APPROVED** [review]
LGTM (tiny style problem could enhance later)

**Pull request merged**